lib/etcconf/hosts.go

```diff
@@ -68,6 +68,14 @@ func (w *hostsWalker) Next() string {
 	return s
 }
 
+func NewHosts(path string) Hosts {
+	if path == "" {
+		path = hostsPath
+	}
+
+	return &hosts{path: path, hosts: make(map[string]net.IP)}
```

Comment:
Might be a little more reader friendly if there was a newline between variable assignments here.